uptane-standard.md

```diff
@@ -868,32 +873,37 @@ In order to perform partial verification, an ECU SHALL perform the following ste
+4. If this ECU is a primary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** all image metadata.  The primary ECU SHOULD also ensure that the image metadata from the director repository doesn't contain any ECU identifiers for ECUs not actually present in the vehicle.
+5. If instead this ECU is a secondary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** the first image metadata listed that includes this secondary ECU's ECU identifier and hardware ID.
+6. For each image name for which the previous two steps **stored** image metadata, obtain verified image metadata for that image by iterating over each mapping in the repository mapping metadata, in order, and performing the following steps for each mapping:
+    1. If the image name does not match the image paths listed in the mapping, skip to the next mapping.
+    2. For each repository listed in this mapping, if top-level metadata has not been refreshed for that repository using {#refresh_toplevel_metadata} in this iteration of the {#full_verification} procedure, refresh top-level metadata for that repository using the steps in {#refresh_toplevel_metadata}.
+    3. For each repository listed in this mapping, perform the procedure in {{resolve_delegations}}, beginning at the top-level Targets metadata in that repository, to find metadata for the image if it exists, and **store** it if it does.
+    4. Find the first set of at least the mapping's threshold number of repositories, including the Director repository, for which all of the following is true for the stored image metadata for each repository:
+        1. The non-custom Targets metadata (i.e., length and hashes) of the unencrypted image are the same.
+        2. If listed, the "MUST match" custom Targets metadata (e.g., hardware identifier and release counter) are the same.
+        3. Any ECU identifiers, if listed, are the same.
+    5. If there is no such set (of at least the mapping's threshold number of repositories meeting the criteria in the prior step), then check the terminating property on this mapping.  If the terminating value for this mapping is true, then abort the search for image metadata for this image, discarding metadata **stored** for this image, as there is no verifiable update for it, and proceed to the next image for which there is **stored** image metadata, if any.  If the terminating value for this mapping is false, continue the search for verified image metadata for this image by continuing to the next mapping, discarding the image metadata for this image from the repositories other than the Director repository.
+7. The image metadata **stored** for each image is now verified.
```

Comment:
I don't like calling it the "image metadata", although I know saying "targets metadata for this image" is clunkier. Do we call it as such elsewhere?

Reply:
Yes, the [hashes, length, custom metadata, etc.] that describes an image is called image metadata in many places elsewhere in the standard.  "targets metadata for this image", in addition to being a little clunkier, might suggest a full targets role on quick read.  I'm fine with whatever's clearest and most consistent across the document -- my current take is that that's "image metadata".

Reply:
This might merit a nice, clear definition somewhere and then a check through the document to make sure that it's called "image metadata" consistently.  (If that's what we agree on....)

Reply:
Is this line even necessary? It's ambiguous: does it mean the image metadata has been verified by the time you reach this step, or does it mean it's up to the implementation to verify it as part of step 7?

Reply:
It was intended to describe that state of things by the time you reach that step.  You're right: it fails at helping to clear things up.  If we think that clarification is helpful, something like `Following the prior steps results in ... ` would probably work better.

Reply:
Removed from text.